gatsby_fp/src/components/Layout/Body/Fields/IntellCont/SubHeading.js

```diff
@@ -2,7 +2,9 @@ import React from "react"
 const shortid = require("shortid")
 
 export default function IntellCont({ intellContList, publicationType }) {
-
+  // Component used for the subheadings under selected publications
```

Comment:
When describing an entire function in a comment, we usually put that comment before the function declaration. (The same idea applies to any instance where a comment describes multiple subsequent lines: it comes before _all_ relevant lines.)